### PR TITLE
feat: added header.

### PR DESCRIPTION
**Pull request by Sahil2004**
# Description

Added the header/navbar of the website.

# Reference

Normal state:
image
Hover state:
image

### CODE REVIEW

File: css/styles.css
Comment (ccetacm) [inline css/styles.css:1]: please use 2 spaces for indentation in html and css files and push in a new commit

**Review by ccetacm — CHANGES_REQUESTED** [review]

File: css/styles.css
Comment (Sahil2004) [reply css/styles.css:1]: Fixed.

**Review by ccetacm — CHANGES_REQUESTED** [review]
please review the design again. the nav elements should be uppercase now

**Comment by Sahil2004** [comment]
> please review the design again. the nav elements should be uppercase now

Done.

**Pull request closed without merging**